### PR TITLE
Clean up Team page and homepage intro

### PR DESCRIPTION
**Pull request by ErikBoesen**
We're losing a lot of people this summer. ¯\\\_(ツ)_/¯

### CODE REVIEW

File: _pages/team.html [hunk 1]
```diff
@@ -6,140 +6,47 @@
...
-            <img src="/assets/img/team/stephen-hoag.png">
-            <h5>Stephen Hoag</h5>
-            <p>Stephen is a new member of the team who works on updating our website.</p>
+            <p>Addison leads our scouting team and helps with outreach.</p>
         </div>
         <div>
             <img src="/assets/img/team/tristan-charpentier.png">
             <h5>Tristan Charpentier</h5>
-            <p>Tristan is one of our team programmers. He is also working on the server of <a href="https://github.com/frc1418/VictiScout">VictiScout</a>.</p>
+            <p>Tristan is an experienced programmer.</p>
```
Comment (Twinters007) [inline _pages/team.html]: Maybe say something about vision? Idk if he's gonna do it.

**Comment by ErikBoesen** [comment]
Removed team leader label from myself, per our conversation

**Comment by ErikBoesen** [comment]
Any further complaints?

**Comment by ErikBoesen** [comment]
I'm going to assume we're good to go here.

**Pull request merged**